### PR TITLE
Add Responsible Disclosure Policy

### PR DESCRIPTION
**Pull request by thibault-martinez**

### CODE REVIEW

File: .github/SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Responsible Disclosure Policy
+
+At the IOTA Foundation, we consider the security of our systems a top priority. Nevertheless, no matter how much effort we put into system security, there can still be vulnerabilities present. If you've discovered a vulnerability, please follow the guidelines below to report it to our security team:
```
Comment (DaughterOfMars) [inline .github/SECURITY.md]: Perhaps this is a better, less repetitive phrasing:

`However; despite our best efforts there may still be vulnerabilities present.`

**Review by Thoralf-M — DISMISSED** [review]

File: .github/SECURITY.md (hunk 1 above)
Comment (thibault-martinez) [reply .github/SECURITY.md]: Done

File: .github/SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+- The scope of the program is limited to technical vulnerabilities in IOTA Foundations' web applications and open source software packages distributed through GitHub. Please do not try to test physical security or attempt phishing attacks against our employees, and so on.
+- Out of concern for the availability of our services to all users, please do not attempt to carry out DoS attacks, leverage black hat SEO techniques, spam people, or other similarly questionable things. We also discourage the use of any vulnerability testing tools that automatically generate significant volumes of traffic.
+
+## What We Promise:
+
+- We will respond to your report within 3 business days with our evaluation of the report, and an expected resolution date.
+- If you have followed the instructions above, we will not take any legal action against you in regard to the report.
+- We will keep you informed during all stages of resolving the problem.
+- To show our appreciation for your effort and cooperation during the report, we will list your name, and a link to a personal website/social network profile on the page below.  This way the public can know you've helped keep the IOTA Foundation secure.
```
Comment (DaughterOfMars) [inline .github/SECURITY.md]: Specify that we will only do this if they want us to?

**Review by Thoralf-M — DISMISSED** [review]

File: .github/SECURITY.md (hunk 2 above)
Comment (thibault-martinez) [reply .github/SECURITY.md]: Done

**Review by DaughterOfMars — DISMISSED** [review]

File: .github/SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+- Out of concern for the availability of our services to all users, please do not attempt to carry out DoS attacks, leverage black hat SEO techniques, spam people, or other similarly questionable things. We also discourage the use of any vulnerability testing tools that automatically generate significant volumes of traffic.
+
+## What We Promise:
+
+- We will respond to your report within 3 business days with our evaluation of the report, and an expected resolution date.
+- If you have followed the instructions above, we will not take any legal action against you in regard to the report.
+- We will keep you informed during all stages of resolving the problem.
+- To show our appreciation for your effort and cooperation during the report, we will list your name, and a link to a personal website/social network profile on the page below, if that is something you would be interested in.  This way the public can know you've helped keep the IOTA Foundation secure.
```
Comment (Thoralf-M) [inline .github/SECURITY.md]:
```suggestion
- To show our appreciation for your effort and cooperation during the report, we will list your name, and a link to a personal website/social network profile on the page below, if that is something you would be interested in. This way the public can know you've helped keep the IOTA Foundation secure.
```

**Review by Thoralf-M — APPROVED** [review]

**Review by DaughterOfMars — APPROVED** [review]

**Review by kwek20 — APPROVED** [review]

**Pull request merged**